projects/core/src/occ/services/occ-endpoints.service.ts

```diff
@@ -183,4 +313,13 @@ export class OccEndpointsService {
         : endpointConfig?.[DEFAULT_SCOPE]) || endpoint
     );
   }
+
+  private validatePrefixConfig() {
```

Comment:
I would avoid mutations on configuration. Let's create method that returns perfix with `/` and replace every usage of `this.config.backend.occ.prefix` with it. Similarly in the other class.

Reply:
I do agree on this, mutating it feels dirty.